collection.c

```diff
@@ -1134,7 +1135,11 @@ static void php_mongo_collection_insert(zval *z_collection, zval *document, zval
 		RETURN_FALSE;
 	}
 
-	if (php_mongo_api_connection_supports_feature(connection, PHP_MONGO_API_WRITE_API)) {
+	is_acknowledged = is_gle_op(z_collection, z_write_options, &link->servers->options TSRMLS_CC);
```

Comment:
I like that you picked ``is_acknowledged`` and not ``is_gle_op``. I was going make sure after reading the truth table in the ticket, but you already did it :-)